docs/sources/panels-visualizations/query-transform-data/transform-data/index.md

```diff
@@ -1007,6 +1007,25 @@ This transformation is available in Grafana 10.1+ as an alpha feature.
 
 Use this transformation to format the output of a time field. Output can be formatted using (Moment.js format strings)[https://momentjs.com/docs/#/displaying/]. For instance, if you would like to display only the year of a time field the format string `YYYY` can be used to show the calendar year (e.g. 1999, 2012, etc.).
 
+### Format string
+
+{{% admonition type="note" %}}
+This transformation is available in Grafana 10.2+ and can be enabled using the `formatString` feature toggle.
+{{% /admonition %}}
```

Comment:
One thing I missed here; we don't need to say this is available in the 10.2 docs because this will first appear in the 10.2 docs. Is this available in Cloud (and thus requires Support to enable the feature for Cloud)? If so there's some standard language I can add here for that. 
```suggestion
{{% admonition type="note" %}}
This transformation can be enabled using the `formatString` feature toggle.
{{% /admonition %}}
```

Reply:
@imatwawana yes, this should be available for cloud aswell!